.github/workflows/add-to-project.yml

```
@@ -0,0 +1,17 @@
+name: Add bugs to bugs project
```

Is this name correct?
```suggestion
name: Add bugs to fluent project
```

---

It should be so, thanks.